puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/tests/katello-pr-test.yaml

```diff
@@ -9,6 +9,10 @@
     triggers:
       - github_pr_pipeline:
           context: 'katello'
+          excluded_regions:
+            - ^webpack\/.+
+            - ^app\/assets\/.+
+            - package.json
```

Comment:
@jlsherrill @waldenraines -- This represents folders that if they are the only things changed will not trigger a build. In this case, the idea being these do not trigger the re-designed katello-pr-test for ruby tests. Can you check accuracy?

Reply:
One question, with the recent change we made that requires the katello job to run, will this cause unmergable prs?

Reply:
That's a fair question, testing this out it appears that the way the included/excluded work is by not triggering a job which would break that required logic. Which kinda sucks. The alternatives are:

 1) Single katello-pr-test job that doe sits own checking for git changes and runs subsets of test tasks based on those changes thus reporting status still just on different aspects
 2) Split pr jobs that do their own checking for git changes and either report success if nothing to run or run the tests and let them dictate status

Option #1 would mean things stay as is, with [test katello] and it runs whatever is needed each time. Option #2 means more status granularity and ability to individually re-run a status.

Reply:
I don't think we do. @waldenraines Can you comment towards the nature of the webpack Ui tests?

Reply:
We don't have any API integration tests, all of the API calls are mocked.  If you changed the API without updating the UI, no tests would fail.  This is likely something we should look at adding in the future.  At the moment we're just relying on the dev to realize that they also need to update the UI if they change the API.

Reply:
i guess I have a slight preference with 1) as it would potentially reduce some re-test time if there were updates in both places.